Tools/clinic/clinic.py

```diff
@@ -1834,7 +1841,8 @@ def is_stop_line(line):
             if self.is_start_line(line):
                 break
 
-        output = output_output()
+        output: str | None = output_output()
+        assert isinstance(output, str)
```

Comment:
This feels a little bit clumsy. I'm not sure if there's a better way, though.

Reply:
Try this:
```
output: str | None
output: = output_output()
```

Reply:
Doh, of course! Thanks!